### PR TITLE
Some updates

### PR DESCRIPTION
**Pull request by chenrui333**
- docs: bump cache action to v3
- review-cask-pr: add .ruby-version and update workflow

### CODE REVIEW

**Comment by Bo98** [comment]
`review-cask-pr` uses `brew ruby` so it shouldn't require any external Ruby.

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+ruby 3.2.2
```
Comment (MikeMcQuaid) [inline .tool-versions:1]: What tool consumes these files?

**Review by MikeMcQuaid — COMMENTED** [review]
Looks good other than one question.

File: .tool-versions (hunk 1 above)
Comment (chenrui333) [reply .tool-versions:1]: the setup-ruby would consume this (i had some issue putting it into review-cask-pr folder, which is kind of a bug in setup-ruby action)

Comment (chenrui333) [reply .tool-versions:1]: gonna merge for now, we can discuss/revisit if needed

**Pull request merged**